cea/default.config

```diff
@@ -52,6 +52,42 @@ archetypes.type = MultiChoiceParameter
 archetypes.choices = comfort, architecture, HVAC, internal-loads, supply, restrictions
 archetypes.help = List of input databases to create.
 
+[streets-helper]
+bbox =
+bbox.type = ListParameter
+bbox.help = Default: empty, it uses district shapefile as bounding box, otherwise indicate Bounding box coordinates. bbox = min Longitude , min Latitude , max Longitude , max Latitude, if empty we
+
+streets = all_private
+streets.type = ChoiceParameter
+streets.choices = all_private, walk, bike, drive, drive_service, all
+streets.help = type of streets to gather. Default All_private
```

Comment:
These help texts show up in the dashboard interface. They need to be checked for grammar / spelling.

Reply:
fixed, thanks!